test/unit/resourceFilter.js

```diff
@@ -1,236 +1,37 @@
-"use strict";
+import assert from "assert";
+import { Filter } from "../../src/resource_filter";
 
-const assert = require("assert");
+import setupDb from "../integration/setup";
 
 describe("resourceFilter", () => {
-  const Filter = require("../../src/resource_filter");
-
-  describe("getExcluded", () => {
```

Comment:
ℹ️ This used to test implementation details and wasn't actually helpful: _what_ we filter is much more important than _how_ we filter.